src/views/HomeView.vue

```diff
@@ -1,11 +1,40 @@
 <template>
   <div class="container">
     <h1 class="text-3xl">Home Page</h1>
+    <div class="mt-10 mx-auto p-10 bg-white border rounded-lg shadow-sm shadow-black">
+      <h3>Hi, {{userLogin}}</h3>
+    </div>
   </div>
 </template>
 
 <script>
-export default {};
+import axios from 'axios'
+
+export default {
+  name: 'HomeView',
+  data() {
+    return {
+      userLogin: 'User is not logged in'
+    };
+  },
+  computed: {
+    userToken() {
+      return this.$store.state.token;
+    }
+  },
+  async created() {
+    const response = await axios.get('/api/v1/users', {
+      headers: {
+        Authorization: 'Bearer' + this.testToken
```

Comment:
this.testToken dptny drimana, Bang ?

Reply:
Sorry bang itu lupa ku ubah

Reply:
Sipp
Tpi kekny emg harus kita minta deh ma BE, sbnarny bisa aja si di store ke local storage informasi ini
![image](https://user-images.githubusercontent.com/66128324/173271054-456c0e95-28a7-44cc-8c0e-37c55b3189fc.png)
tpi keknya kurang aman (?)

Yg di video, yg disimpan apa aja y, Bang ? token aja kah?

Reply:
token aja kayaknya bang, Jadi klo mw liat usernya yang login perlu minta ke API nya lagi pake tokennya bang

Reply:
Sipp brarti tinggal itu, kan...nanti gmeet dh diskusikan lgi ke BE, ini aku approve aj y

Reply:
ok bang